usr/sbin/truenas-install

```diff
@@ -282,8 +288,14 @@ install_loader()
 	    chroot ${_mnt} grub-install --target=i386-pc /dev/${_disk}
 
 	    echo "Stamping EFI loader on: ${_disk}"
-	    chroot ${_mnt} mkdosfs -F 32 -s 1 -n EFI /dev/${_disk}2
+        # Check if nvme disk and use different naming convention.
+        if [[ "${_disk}" == *"nvme"* ]]; then
+	    chroot ${_mnt} mkdosfs -F 32 -s 1 -n EFI /dev/${_disk}p2
+	    chroot ${_mnt} mount -t vfat /dev/${_disk}p2 /boot/efi
+        else
+        chroot ${_mnt} mkdosfs -F 32 -s 1 -n EFI /dev/${_disk}2
```

Comment:
Can we please fix the indentation and not have a mix of tabs/spaces ?
Also how about using a single variable reducing duplication ?

Reply:
Ok I just cleaned it up, is it looking better?